irods/auth/pam_password.py

```diff
@@ -32,11 +32,16 @@ def _authenticate_pam_password(conn, req):
     """
     _logger.debug('----------- %s (begin)', _scheme)
 
-    # By design, we persist this "depot" object over the whole of the authentication
-    # exchange with the iRODS server as a means of sending password information to the
-    # native phase of that process.  This has to be done in a way that preserves
-    # the current environment (i.e. not writing to .irodsA) in the event that we
-    # are authenticating without the help of iCommand-type client env/auth files.
+    # Next, create and persist a "depot" object over the whole of the authentication
+    # exchange with the iRODS server.
+    #
+    # This is done as a means of sending password information to the native phase
+    # of authentication (with an appropriate token generated by the server as a
+    # native password input).   This must be done in a way that preserves the
```

Comment:
There are 3 spaces following the period (.).

Reply:
Done.